docs/development_environment.mdx

```diff
@@ -125,6 +125,11 @@ Each time you start a new terminal session, you will need to activate your virtu
 source venv/bin/activate
 ```
 
+Some components require that the translations are generated properly. In some cases this is the only way to pass the automated tests. For example for the script component, run the following:
```

Comment:
I think it might make more sense to investigate how we can automatically handle this.